app/schema/person/update.js

```diff
@@ -0,0 +1,22 @@
+const Joi = require('joi')
+
+const schema = Joi.object({
+  personReference: Joi.string().required(),
+  firstName: Joi.string().required(),
+  lastName: Joi.string().required(),
+  dateOfBirth: Joi.date().iso().optional().allow('').allow(null),
+  address: Joi.object({
```

Comment:
Do we need max length checks on the fields or are you happy for the DB to catch that?

Reply:
Happy for the DB to catch that at the moment. Not a public facing API so probably not worth doing lots of work on errors.